TARGETS

```diff
@@ -446,6 +453,19 @@ cpp_library(
     external_deps = ROCKSDB_EXTERNAL_DEPS,
 )
 
+cpp_library(
+    name = "rocksdb_third_party_gtest",
+    srcs = ["third-party/gtest-1.8.1/fused-src/gtest/gtest-all.cc"],
+    headers = [u'third-party/gtest-1.8.1/fused-src/gtest/gtest.h'],
```

Comment:
I think `u'<string>'` is not allowed.

Reply:
> Thanks @pdillinger for the PR. Mostly LGTM. Could you run `python buckifier/buckify_rocksdb.py`?

Considering I've modified the generation code and there are corresponding changes to TARGETS, I obviously did, but who knows what's different in the GitHub build. Maybe it's just the problem with the existing implementation of add_library generating a u string for headers.